sweddit_combined/decisions/ChineseFormables.txt

```diff
@@ -154,19 +87,46 @@ country_decisions = {
 				}
 				add_permanent_claim = IMP
 			}
-			add_prestige = 25
+			add_prestige = 50
```

Comment:
Reason behind changing this?

Reply:
Den här skyller jag på annima, det är han som gjort. tar bort